pkg/storage/backend-storage/backend-storage_test.go

```diff
@@ -320,4 +322,75 @@ var _ = Describe("Backend Storage", func() {
 			}),
 		)
 	})
+
+	Context("PVC size", func() {
```

Comment:
Those seem to me more unit tests rather the functional ones. Can we convert these to UT?

Reply:
These are UT already

Reply:
Sorry, forget it, I don't know why I read this as under tests